server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/remote/delivery/MailDelivrerToHost.java

```diff
@@ -66,7 +67,7 @@ public ExecutionResult tryDeliveryToHost(Mail mail, InternetAddress[] addr, Host
             transport = (SMTPTransport) session.getTransport(outgoingMailServer);
             transport.setLocalHost(props.getProperty("mail.smtp.localhost", configuration.getHeloNameProvider().getHeloName()));
             connect(outgoingMailServer, transport);
-            transport.sendMessage(adaptToTransport(mail.getMessage(), transport), addr);
+            transport.sendMessage(adaptToTransport(mail.getMessage(), transport), toArray(addr));
```

Comment:
well, `addr.toArray(new InternetAddress[0])` is the idiomatic way to do that. Does it deserve a submethod?